docs/api-composite.md

```diff
@@ -66,3 +67,5 @@ Returns **Sharp**
 [6]: https://www.npmjs.org/package/color
 
 [7]: https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Error
+
+[8]: https://jcupitt.github.io/libvips/API/current/libvips-conversion.html#VipsBlendMode
```

Comment:
libvips has moved to its own GitHub org so these links now start `https://libvips.github.io/libvips/...`